### DISCUSSION TITLE
A Website - Let's Get the Ball Rolling

### DISCUSSION THREAD

**Discussion by kjaymiller** [BlackPythonDevs/blackpythondevs#13] [General]
There are a lot of things that have to be figured out if we were to do a website. I'm totally down but lets get them figured out together:

Feel free to add your thoughts to any of these.

- [ ] Project Maintainers
- [ ] Stack
- [ ] Voice
- [ ] Design
- [ ] Accessibility (A11y)
- [ ] Localization (L18n)

**Comment by oleksis** [comment]
## Stack

Statics sites generators 

- [x] [Jekyll](https://jekyllrb.com/) (built in GitHub Pages - Ruby)
- [ ] [MkDocs ](https://www.mkdocs.org/) (Python)
- [ ] [Render Engine](https://render-engine.readthedocs.io/en/latest/) (Python)
- [ ] [Hugo](https://gohugo.io/) (Go)
- [ ] 

If you add others in the comments I'll add it to the list

**Reply by kjaymiller** [reply]
Shameless plug for my own (static site generator - Render Engine)[https://github.com/render-engine/render-engine) Which is also in Python.

If this was something that folks wanted _professional_ experience with contributing to. I would suggest something dynamic and hosted (we can do azure and I can likely host for free - I'm not using hardly any of my monthly credit.

**Reply by kjaymiller** [reply]
This is not a dig on ssg's not being professional but more of a (if folks wanting to get into webdev with Python were looking for a stack they can share on a resume something like Django or FastAPI may be more appealing.

**Comment by oleksis** [comment]
## I18n
- [x] Jekyll (https://kitian616.github.io/jekyll-TeXt-theme/docs/en/i18n , https://mincong.io/en/jekyll-i18n/)
- [ ] MkDocs (https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/)
- [ ]

**Comment by oleksis** [comment]
## Content for the website: 
- [x] About Us
- [x] Events: Meetings, CFPs
- [x] Conferences
- [x] Community (Link to rules for joining Discord and joining link)

**Reply by kjaymiller** [reply]
I like this direction but we simply don't have a lot of this. 

I think we can start closer to this:

- [x] Mission
- [x] About Us
- [x] Events: Meetings, CFPs
- [x] Conferences (I'm separating this because I think we can make this a blog or something that shows information but also people that are speaking at the event.)
- [ ] Community (Link to rules for joining Discord and joining link)
- [ ] Contact Us

**Comment by oleksis** [comment]
Static Site Generators [¹][1]

## Fronted
- Pico / Bootstrap / Tailwind / Materialize (CSS)
- Typography (Awesome fonts)
- Iconography ( .ico, .png, .svg )
- Internationalization (I18n)
- Jinja2 ,  Liquid (Template / Layout)
- Markdown (Content)
- 

## Backend
- Jekyll 
- MkDocs
- Render Engine 
- Next.js
- Docusaurus
- Hugo
- 

[🗳️ Please Vote](https://github.com/BlackPythonDevs/.maintainers/discussions/14)

[1]: https://jamstack.org/generators/

**Comment by oleksis** [comment]
## CSS

To customize the Theme we will use the CSS framework:

- [ ] [Tailwind CSS](https://tailwindcss.com/)
- [ ] [Primer](https://primer.style/)
- [ ] [Materialize](https://materializecss.com/)
- [x] [Pico](https://picocss.com/docs)
- [ ] Other

**Comment by oleksis** [comment]
Background images: 

Black Python Devs: https://x.com/oleksis/status/1765411739036786736?s=20

**Comment by oleksis** [comment]
Fonts: 
- Modern Society: https://www.dafontfree.io/modern-society-free-font/

**Comment by brightcemeka** [comment]
Here's a UX Audit for the current website.

[Black Python Devs UX Audit.pdf](https://github.com/BlackPythonDevs/.maintainers/files/14589708/Black.Python.Devs.UX.Audit.pdf)

**Reply by oleksis** [reply]
Good and amazing Job 👏🏽🎉

**Comment by oleksis** [comment]
# Color Palette

> grabbed from the website on light mode

- [Palette.pdf](https://github.com/BlackPythonDevs/.maintainers/files/14755209/Palette.pdf)
